.github/workflows/flatpak.yaml

```diff
@@ -0,0 +1,30 @@
+on: 
+  push:
+    tags:
+      - '*'
+workflow_dispatch
+
+jobs: 
+  flatpak:
+     
+    runs-on: ubuntu-20.04
+    
+    steps:
+    - uses: actions/checkout@v2 
+    - name: Install Flatpak
+      run: >
+        sudo apt install flatpak && flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

Comment:
`flatpak-builder also needs to be installed:
```
sudo apt-get install flatpak flatpak-builder
```

Reply:
I changed it to having apt install the flatpak-builder instead of flathub now.